src/legacy/core_plugins/kibana/public/home/components/home.js

```diff
@@ -64,6 +64,7 @@ export class Home extends Component {
       isNewKibanaInstance: false,
       isWelcomeEnabled,
       showTelemetryDisclaimer,
```

Comment:
I'm wondering if we can remove this property now from the state. I couldn't find any other usage of it (although I may have not run the right search).

Reply:
I'll take a look and see if we can remove it

Reply:
I removed 'showTelemetryDisclaimer' [here](https://github.com/elastic/kibana/pull/53084/commits/bf82c8395056eb07d10ec566c76ce02204e1cb67#diff-7127028f4c0b8337b65f806cd35e8023R53)